### ISSUE TITLE
Deadmaster/Reality Bending interactions

### ISSUE THREAD

**Issue by ManBroCalrissian** [SolastaMods/SolastaUnfinishedBusiness#3646]
Game Version: 1.5.92
UB Version: 1.5.92.2

TLDR: Reality Bending is inconsistent when summoning is compared to damage dice nominal output

I have previously tested **R**eality **B**ending (RB) from College of Life extensively and found the additive dice in the nominal damage output to be accurate when base cast (RB=2) and upcast (RB=2+(upcast spell lvl - spell lvl)).  Now to the bug:

When casting summons at base spell level, the recent fix makes RB add an additional summon, but casting at base spell level does not remove the concentration requirement. This may be intended behavior as RB says "Necromancy spells are cast at a spell slot 2 levels higher" and the Deadmaster Scatter ability says "If upcasted, you don't require concentration to raise dead." Not exactly the same terminology.

It is established that RB makes summon spells 2 slots higher for determining the number of summons but that RB spells are not necessarily considered to be "upcast" specifically. With that being said, If you upcast an RB summon spell 1 level higher, it only summons 1 minion. You do not get the additional summon. If upcast 2 levels higher, it creates 2 total minions. 

According to my understanding of how Reality Bending works with damage dice, the wording on summon abilities and spells indicates that RB summon spells should always summon two minions and that a 2 level higher upcast should summon 3 minions, etc. The concentration requirement is gray area and may be intended behavior.

Attached file has a level 9 Deadmaster/3 College of Life and a level 12 Deadmaster.
[RBDM.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/11860670/RBDM.zip)

**Comment by ThyWoof** [comment]
Buddy, I have a huge attention problem. Add to that the fact I speak many languages, all of them very badly :-)

in summary: You expect 3 summons instead of 2 when upcasting let’s say a level 2 raise dead with a level 4 slot but you only get 2, correct?

**Comment by ThyWoof** [comment]
One sentence on answer please LOL.

**Comment by ManBroCalrissian** [comment]
I'm not really sure what to expect. It just doesn't make sense that base level gives 2 summons but 1 level upcast gives 1 summon.

Sorry for the long post, but I didn't know how to thoroughly explain the the inconsistencies. Just trying to help. (8

**Comment by ThyWoof** [comment]
Ok, now I understood. Will check.

**Comment by ThyWoof** [comment]
![image](https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/13087646/9c88abcf-d4ad-42d8-bd8a-9676bff09d43)
